google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -218,7 +224,87 @@ public static Value pgJsonb(@Nullable String v) {
   }
 
   /**
-   * Returns a {@code BYTES} value.
+   * Return a {@code PROTO} value.
+   *
+   * @param v Not null Proto message.
+   */
+  public static Value protoMessage(AbstractMessage v) {
+    Preconditions.checkNotNull(
+        v, "Use protoMessage((ByteArray) null, MyProtoClass.getDescriptor()) for null values.");
```

Comment:
Can we do better? If not, please add a comment.

Reply:
Modified the function doc to specify the method is specifically for Not Null messages.